src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -123,6 +123,10 @@ public EventTypeService(final EventTypeRepository eventTypeRepository,
 
     public void create(final EventTypeBase eventType) throws TopicCreationException, InternalNakadiException,
```

Comment:
throws `DbWriteOperationsBlockedException`
Same for other methods in the class

Reply:
Fixed